TakafulDAO/takaturn
# feat(deployment): support private network

No changes in the contracts

Add configs for a private network deployed by Khalid and staging tests against that network, and minor refactor on deploy scripts

Tests made:
- check initialization variables
- create new term
- join term with yield generation
- start term
- pay contribution
- close funding period
- start new cycle
- withdraw fund
- withdraw collateral

Also added some mocks of the zaynfi contracts needed and their deploy scripts.

cc: @kkharji is tagged so he can check what he needs
